accounts/urls.py

```diff
@@ -1,10 +1,11 @@
 from django.urls import path
-from .views import RegistrationViewAPI, GenerateCodeAPI, VerifyCodeAPI
+from .views import RegistrationViewAPI, GenerateCodeAPI, VerifyCodeAPI, LogoutView
```

Comment:
The logout view? I don't think we need it. I have enabled blacklist token app that still in-progress.